packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -982,7 +982,7 @@ class PhysicalModel extends SingleChildRenderObjectWidget {
 ///
 /// See also:
 ///
-///  * [ShapeBorderClipper], which converts a [ShapeBorder] to a [CustomerClipper], as
+///  * [ShapeBorderClipper], which converts a [ShapeBorder] to a [CustomClipper], as
```

Comment:
LOL. What exactly does a "Customer Clipper" do?  Doesn't sound like something we'd want to implement. :-)

Reply:
Haha!